internal/model/release_values.go

```diff
@@ -0,0 +1,250 @@
+	Edition                       string      `yaml:"edition,omitempty"`
+	AcceptLicenseAgreement        string      `yaml:"acceptLicenseAgreement,omitempty"`
+	OfflineMaintenanceModeEnabled bool        `yaml:"offlineMaintenanceModeEnabled,omitempty"`
+	Resources                     Resources   `yaml:"resources,omitempty"`
+	Labels                        interface{} `yaml:"labels,omitempty"`
+}
+type Requests struct {
+	Storage string `yaml:"storage,omitempty"`
+}
+type MatchLabels struct {
+	App                    string `yaml:"app,omitempty"`
+	HelmNeo4JComVolumeRole string `yaml:"helm.neo4j.com/volume-role,omitempty"`
+}
+type SelectorTemplate struct {
+	MatchLabels MatchLabels `yaml:"matchLabels,omitempty"`
+}
+type Selector struct {
+	StorageClassName string           `yaml:"storageClassName,omitempty"`
+	AccessModes      []string         `yaml:"accessModes,omitempty"`
+	Requests         Requests         `yaml:"requests,omitempty"`
+	SelectorTemplate SelectorTemplate `yaml:"selectorTemplate,omitempty"`
+}
+type DefaultStorageClass struct {
+	AccessModes []string `yaml:"accessModes,omitempty"`
+	Requests    Requests `yaml:"requests,omitempty"`
+}
+type Dynamic struct {
+	StorageClassName string   `yaml:"storageClassName,omitempty"`
+	AccessModes      []string `yaml:"accessModes,omitempty"`
+	Requests         Requests `yaml:"requests,omitempty"`
+}
+type Volume struct {
+	SetOwnerAndGroupWritableFilePermissions bool `yaml:"setOwnerAndGroupWritableFilePermissions,omitempty"`
+}
+type VolumeClaimTemplate struct {
+}
+type Data struct {
+	Mode                string              `yaml:"mode,omitempty"`
+	Selector            Selector            `yaml:"selector,omitempty"`
+	DefaultStorageClass DefaultStorageClass `yaml:"defaultStorageClass,omitempty"`
+	Dynamic             Dynamic             `yaml:"dynamic,omitempty"`
+	Volume              Volume              `yaml:"volume,omitempty"`
+	VolumeClaimTemplate VolumeClaimTemplate `yaml:"volumeClaimTemplate,omitempty"`
+}
+type Share struct {
+	Name string `yaml:"name,omitempty"`
+}
+type Backups struct {
+	Mode  string `yaml:"mode,omitempty"`
+	Share Share  `yaml:"share,omitempty"`
+}
+type Logs struct {
+	Mode  string `yaml:"mode,omitempty"`
+	Share Share  `yaml:"share,omitempty"`
+}
+type Metrics struct {
+	Mode  string `yaml:"mode,omitempty"`
+	Share Share  `yaml:"share,omitempty"`
+}
+type Import struct {
+	Mode  string `yaml:"mode,omitempty"`
+	Share Share  `yaml:"share,omitempty"`
+}
+type Licenses struct {
+	Mode  string `yaml:"mode,omitempty"`
+	Share Share  `yaml:"share,omitempty"`
+}
+type Volumes struct {
+	Data     Data     `yaml:"data,omitempty"`
+	Backups  Backups  `yaml:"backups,omitempty"`
+	Logs     Logs     `yaml:"logs,omitempty"`
+	Metrics  Metrics  `yaml:"metrics,omitempty"`
+	Import   Import   `yaml:"import,omitempty"`
+	Licenses Licenses `yaml:"licenses,omitempty"`
+}
+type NodeSelector struct {
+}
+type Annotations struct {
+}
```

Comment:
empty struct ?

Reply:
i dont see Annotations in 4.4 values.yaml ...which annotations are these ? PodSpec ? statefulset ?